SpotifyAPI/Web/SpotifyWebAPI.cs

```diff
@@ -1911,51 +1826,161 @@ public Paging<T> GetPreviousPage<T>(Paging<T> paging)
             return await GetPreviousPageAsync<Paging<T>, T>(paging);
         }
 
+        private ListResponse<bool> DownloadDataAltAndExtract(string url)
```

Comment:
How about making this little more generic, I would suggest:
`private ListResponse<T> DownloadList<T>(string url)`

Same for other methods